api/gateway/v1alpha1/gateway_types.go

```diff
@@ -202,6 +204,47 @@ func (gw *Gateway) Validate(ctx context.Context, kube kclient.Reader) error {
 				return fmt.Errorf("interface %s IP %s must be an IPv4 address: %w", name, ifaceIP, ErrInvalidGW)
 			}
 		}
+
+		if iface.PCI != "" {
```

Comment:
Dataplane will validate that too (see https://github.com/githedgehog/dataplane/blob/bc4c6685cd6f15a365f9b49ef8e3b28f0624236c/hardware/src/pci/address.rs#L156).

It's fine if both validate, but we should then make sure that the validations are aligned in that the CRD one should not be more restrictive than what dataplane can support.